src/SeleniumLibrary/keywords/alert.py

```diff
@@ -76,7 +78,7 @@ def alert_should_be_present(
 
     @keyword
     def alert_should_not_be_present(
-        self, action: str = ACCEPT, timeout: Union[float, str, None] = 0
+        self, action: str = ACCEPT, timeout: Union[timedelta] = 0
```

Comment:
This is wrong because an argument with type `timedelta` cannot really have an integer as a default. You probably also meant `Optional[timedelta]` not `Union[timedelta]` (but I'd leave `Optional[]` away as well).

Reply:
Hmmm, what would be suitable default? The easiest would be to change it to `None` but that would be somewhat backwards incompatible (Because it would default to library default timeout.) Would: 
```
def alert_should_not_be_present(self, action: str = ACCEPT, timeout = timedelta(seconds=0);
```
be better?